Controller/FolderController.php

```diff
@@ -29,19 +29,13 @@ class FolderController extends Controller
      * @throws NotFoundHttpException $location is flagged as invisible
      * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function showFolderListAsideViewAction( Location $location )
+    public function showFolderListAsideViewAction( Location $location, $viewType, $layout = false, array $params = array() )
     {
-        $response = new Response();
-
         if ( $location->invisible )
         {
             throw new NotFoundHttpException( "Location #$location->id cannot be displayed as it is flagged as invisible." );
         }
 
-        $response->setSharedMaxAge( $this->getConfigResolver()->getParameter( 'content.default_ttl' ) );
```

Comment:
it will be cached because now we are using ``` $this->get( 'ez_content' )->viewLocation```, and that will take care of it. 
if we don't use viewLocation as it is here know then we will need to add cache directives.